content/source/docs/enterprise/workspaces/settings.html.md

```diff
@@ -73,11 +73,13 @@ After changing this setting, you must click the "Update SSH key" button below it
 
 If you need to prevent Terraform runs for any reason, you can lock a workspace. This prevents users with write access from manually queueing runs, prevents automatic runs due to changes to the backing VCS repo, and prevents the creation of runs via the API. To enable runs again, a user must unlock the workspace.
 
+Locking a workspace also restricts state uploads. In order to upload state, the workspace must be locked by the user who is uploading state. Note that the atlas backend does not respect this restriction, so state uploads when using the atlas backend are still allowed when the workspace is locked.
+
 Users with write access can lock and unlock a workspace, but can't unlock a workspace which was locked by another user. Users with admin privileges can force unlock a workspace even if another user has locked it.
 
 Locks are managed with a single "Lock/Unlock/Force unlock `<WORKSPACE NAME>`" button. TFE asks for confirmation when unlocking.
 
-~> **Important:** Locking a workspace prevents runs within TFE, but it **does not** prevent state from being updated. This means a user with write access can still modify the workspace's resources by running Terraform outside TFE with [the `atlas` remote backend](/docs/backends/types/terraform-enterprise.html). To prevent confusion and accidents, avoid using the `atlas` backend in normal workflows; to perform runs from the command line, see [TFE's CLI-driven workflow](../run/cli.html).
+~> **Important:** Locking a workspace prevents runs within TFE, but it **does not** prevent state from being updated when using the atlas backend. This means a user with write access can still modify the workspace's resources by running Terraform outside TFE with [the `atlas` remote backend](/docs/backends/types/terraform-enterprise.html). To prevent confusion and accidents, avoid using the `atlas` backend in normal workflows; to perform runs from the command line, see [TFE's CLI-driven workflow](../run/cli.html).
```

Comment:
This seems like a duplicate of the comment above, so I'd suggest we remove the comment above or explicitly refer to this note.

Reply:
agree. I think this comment has a lot more in it than the thing i added above, so maybe we extract bits of this comment to the paragraph I added. I do like that this is called out in an "important" box though.

Reply:
@nfagerlund what do you recommend here? Happy to jump on a zoom to explain the subtleties here.